build.sbt

```diff
@@ -121,8 +121,8 @@ val sharedSettings = Seq(
   crossScalaVersions := Seq("2.11.12", "2.12.8"),
   fork in Test := true, // We have to fork to get the JavaOptions
   libraryDependencies ++= Seq(
-    "org.scalacheck" %% "scalacheck" % "1.13.4" % "test",
-    "org.scalatest" %% "scalatest" % "3.0.0" % "test",
+    "org.scalacheck" %% "scalacheck" % "1.14.0" % "test",
```

Comment:
we need to update scalacheck and scalatest atomically because we have a monorepo, so can you revert this for now?  if there's some reason why we need to update scalacheck and scalatest, we can work on making it happen internally.

Reply:
These are the only versions published for 2.13, unfortunately.

Reply:
oh, we actually already updated to those versions internally, for exactly that reason.  my bad!!  thanks =)